packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -167,6 +175,24 @@ export default async function initSanity(
 
   debug(`Dataset with name ${datasetName} selected`)
 
+  // If user doesn't want to output any template code
+  const bareOutput = args.argv.find((el) => el === '--bare')
+  if (bareOutput) {
+    print(`\n${chalk.green('Success!')} Below are your project details:\n`)
+    print(`Project ID: ${chalk.cyan(projectId)}`)
+    print(`Dataset: ${chalk.cyan(datasetName)}`)
+    print(
+      `\nYou can find your project on Sanity Manage — https://www.sanity.io/manage/personal/project/${projectId}\n`
```

Comment:
Projects can be optionally be attached to an organization, so `personal` is not always correct here. Use the redirect route?
```suggestion
      `\nYou can find your project on Sanity Manage — https://www.sanity.io/manage/project/${projectId}\n`
```

Reply:
Thanks! Wasn't aware 🙇🏽‍♂️